libraries/AC_Autorotation/AC_Autorotation.cpp

```diff
@@ -251,54 +321,121 @@ float AC_Autorotation::get_rpm(bool update_counter)
     return current_rpm;
 }
 
+void AC_Autorotation::initial_flare_estimate()
+{
+    //estimate hover thrust
+    float _col_hover_rad = radians(_col_min + (_col_max - _col_min)*_col_hover);
+    float b = _param_solidity*6.28f;
+    _disc_area=3.14*sq(_param_diameter*0.5f);
```

Comment:
We have a define for PI. 
```suggestion
    _disc_area=M_PI * sq(_param_diameter*0.5);
```

Reply:
done